diff --git a/configure.ac b/configure.ac
index 52adabafaa..7b32e79201 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1953,6 +1953,9 @@ AS_IF([test $enable_fatal_warnings -ne $DISABLED], [
     AC_MSG_NOTICE([Enabling fatal compiler warnings])
     CFLAGS="$CFLAGS $WERROR"
 ])
+
+CFLAGS="$CFLAGS -fplugin=\$(top_builddir)/../fosa/checkargs.so -fplugin-arg-checkargs-store=\$(top_builddir)/fosa-store.txt"
+
 AC_SUBST(CFLAGS)
 
 dnl This is useful for use in Makefiles that need to remove one specific flag
