diff --git a/configure.ac b/configure.ac
index 95adcffb6..2b6720be3 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1846,6 +1846,9 @@ AS_IF([test $enable_fatal_warnings -ne $DISABLED], [
     AC_MSG_NOTICE([Enabling fatal compiler warnings])
     CFLAGS="$CFLAGS $WERROR"
 ])
+
+CFLAGS="$CFLAGS -fplugin=/home/clumens/src/gcc-python-plugin/python.so -fplugin-arg-python-script=/home/clumens/src/fosa/check-args.py -fplugin-arg-python-messages=/home/clumens/src/pacemaker/fns.pickle"
+
 AC_SUBST(CFLAGS)
 
 dnl This is useful for use in Makefiles that need to remove one specific flag
