diff --git a/configure.ac b/configure.ac
index 4b7d09f4e..845963485 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2016,6 +2016,9 @@ AS_IF([test $enable_fatal_warnings -ne $DISABLED], [
     AC_MSG_NOTICE([Enabling fatal compiler warnings])
     CFLAGS="$CFLAGS $WERROR"
 ])
+
+CFLAGS="$CFLAGS -DPCMK__WITH_ATTRIBUTE_OUTPUT_ARGS -fplugin=/home/clumens/src/fosa/findmessages.so -fplugin-arg-findmessages-store=/home/clumens/src/pacemaker/fosa-store.txt"
+
 AC_SUBST(CFLAGS)
 
 dnl This is useful for use in Makefiles that need to remove one specific flag
